### PR TITLE
reaper: Fix hash

### LINKED ISSUE #2470: reaper@5.980: hash check failed
Updating 'reaper' (5.979 -> 5.980)
Downloading new version
reaper5980_x64-install.exe (11.4 MB) [=====================================================================================================================================================================] 100%
Checking hash of reaper5980_x64-install.exe ... ERROR Hash check failed!
App:         extras/reaper
URL:         https://www.reaper.fm/files/5.x/reaper5980_x64-install.exe#/dl.7z
First bytes: 4D 5A 90 00 03 00 00 00
Expected:    4f7c9abc72c5934d2d92378efc51d42a276b3be019f06bb2d620553b19bacabc
Actual:      d11b5d8df8a28e94ddae4f5215440f299782c03fbd191494190da91ef59080d6

### PR DESCRIPTION
**Pull request by niheaven**
Fix #2470

### CODE REVIEW

File: bucket/reaper.json [hunk 1]
```diff
@@ -21,21 +33,9 @@
             "Move-Item \"$dir\\`$INSTDIR`$_8_\\*\" -Destination \"$dir\"",
             "",
             "# Remove unused folders",
```
Comment (Ash258) [inline bucket/reaper.json]: Not needed comment

```suggestion
```

File: bucket/reaper.json [hunk 2]
```diff
@@ -21,21 +33,9 @@
...
-            "hash": "4f7c9abc72c5934d2d92378efc51d42a276b3be019f06bb2d620553b19bacabc"
-        },
-        "32bit": {
-            "url": "https://www.reaper.fm/files/5.x/reaper5980-install.exe#/dl.7z",
-            "hash": "b16fa33572950a330846b014a50a43a8218c4b28b9d8194edc86709e68eea634"
-        }
-    },
     "checkver": {
         "url": "https://www.reaper.fm/download.php",
         "re": "Version (\\d+\\.\\d+)"
```
Comment (Ash258) [inline bucket/reaper.json]:
```suggestion
        "regex": "Version ([\\d.]+):"
```

File: bucket/reaper.json [hunk 3]
```diff
@@ -1,6 +1,18 @@
 {
-    "version": "5.980",
     "homepage": "https://www.reaper.fm",
+    "version": "5.980",
+    "license": "Proprietary",
```
Comment (Ash258) [inline bucket/reaper.json]: Is there some URL which could be provided?

Comment (niheaven) [reply bucket/reaper.json]: I haven't found one, only a `license.txt` attached within package.

**Pull request merged**